elide-model-config/src/test/resources/validator/valid/models/tables/player_stats.hjson

```diff
@@ -20,6 +20,18 @@
       A tag
       with newline
       ''']
+      arguments: [
+          {
+             name: format
+             type: TEXT
+             default: 999999D00
+          }
+          {
+             name: countryCode
```

Comment:
I don't understand the purpose of these table parameters.  If we are going to create a test model with table params, let's make them meaningful.